### DISCUSSION TITLE
posts/how-to-use-fullcalendar-in-next-js/

### DISCUSSION THREAD

**Discussion by dcangulo** [dcangulo/davidangulo.xyz#141] [Announcements]
<h1>posts/how-to-use-fullcalendar-in-next-js/</h1><a href='https://www.davidangulo.xyz/posts/how-to-use-fullcalendar-in-next-js/'>https://www.davidangulo.xyz/posts/how-to-use-fullcalendar-in-next-js/</a>

**Comment by dcangulo** [comment]
<i>This comment was originally posted by <a href='https://disqus.com/by/ka_tam/'>Ka Tam (@ka_tam)</a> on 2019-08-26 20:06:53.</i><br><p>hi Next.js has deprecated multiple export in</p><p><code>dynamic.js:66 The modules option for next/dynamic has been deprecated. See here for more info https://err.sh/zeit/next.js/next-dynamic-modules</code></p><p>It seemed since this warning came out, the multiple css include fails and the Calendar control was back to having no css rendering.</p><p>Next.js posted this sample code, but I am unclear on how to join individual dynamic and render on the final result.<br><a href="#" rel="nofollow noopener" title="#">https://github.com/zeit/next.js/blob/master/errors/next-dynamic-modules.md</a></p>

**Reply by AdventureBear** [reply]
I'm glad i found this article and comment, would love to know if there is a more recent tutorial or fix? Is FullCalendar still recommended for Next.jS?
